app/scripts/views/mixins/signed-out-notification-mixin.js

```diff
@@ -7,38 +7,30 @@
 define(function (require, exports, module) {
   'use strict';
 
-  const BaseView = require('views/base');
   const Notifier = require('lib/channels/notifier');
   const Session = require('lib/session');
+  const Url = require('lib/url');
 
   var Mixin = {
     notifications: {
       // populated below using event name aliases
     },
 
     clearSessionAndNavigateToSignIn () {
-      // Unset uid otherwise it will henceforth be impossible
-      // to sign in to different accounts inside this tab.
-      this.relier.unset('uid');
-      this.relier.unset('email');
       this.user.clearSignedInAccountUid();
-      if (this._formPrefill) {
```

Comment:
Oh my, this is an awesome side-effect that I hadn't considered, formPrefill will be cleared automatically.